config/mastodon.yml

```diff
@@ -0,0 +1,3 @@
+---
+shared:
+  software_update_url: <%= ENV.fetch('UPDATE_CHECK_URL', 'https://api.joinmastodon.org/update-check') %>
```

Comment:
This is sort of odd to have a whole yaml file for just the one value -- however, I'm pretty convinced that are, at minimum, a bunch of values from Mastodon::Version that would make sense in here as well; and I suspect more from throughout the codebase.